### ISSUE TITLE
Newline after SELECT is not being parsed as whitespace

### ISSUE THREAD

**Issue by map222** [GoogleCloudDataproc/spark-bigquery-connector#390]
Hi,
I am running queries using the connector, and I have found that if you run a simple query like:
```
SELECT
<column>
FROM <table>
```

with a newline and no ` `  after the `SELECT` statement, you get an error:
```
1) Error in custom provider, java.lang.IllegalArgumentException: Invalid Table ID 'SELECT
<column>
FROM <table>. Must match '^(((\S+)[:.])?(\w+)\.)?([\S&&[^.:]]+)$$'
```

If I then add a space after the `SELECT`, it will run. 
```
SELECT 
<column>
FROM <table>
```

I found this issue confusing, as the queries without the space will run without error directly in BQ.

**Comment by davidrabinowitz** [comment]
A fix will be added to the next version (See PR #392). In the meantime you can workaround this be applying the following on the read: `df = spark.read.format("bigquery").load(sql.replaceAll("\\s+", " "))`

**Comment by map222** [comment]
Awesome!

**Comment by davidrabinowitz** [comment]
Fixed by PR #392
